any_parser/batch_parser.py

```diff
@@ -8,6 +13,8 @@
 from any_parser.base_parser import BaseParser
 
 TIMEOUT = 60
+MAX_FILES = 1000
```

Comment:
There is no need to restrict on this. For the batch API, the logic is that
- there is a crone job for every 2 hour
- there is a scan every 5 mins and if the batch queue size is more than 1000
One of the two situation match will trigger a batch run.